apps/judicial-system/backend/src/app/modules/case/test/internalCaseController/deliverCaseToPolice.spec.ts

```diff
@@ -41,7 +41,7 @@ describe('InternalCaseController - Deliver case to police', () => {
       const then = {} as Then
 
       await internalCaseController
-        .deliverCaseToPolice(caseId, user, theCase, { userId })
+        .deliverCaseToPolice(caseId, theCase, { user })
```

Comment:
Does the user here need to be wrapped in `{}`? It's already an object, am I misunderstanding something?

Reply:
The deliver DTO contains one field, user, which happens to be an object, so this is correct. We could let the DTO list all the fields of the user object but that would be a bit strange and would require us to change the DTO every time we change the User type.